pkg/services/store/object/dummy/dummy_server.go

```diff
@@ -74,13 +80,26 @@ func (i dummyObjectServer) findObject(ctx context.Context, uid string, kind stri
 
 	getLatestVersion := version == ""
 	if getLatestVersion {
-		objVersion := obj.History[len(obj.History)-1]
-		return obj, objVersion, nil
+		return obj, obj.RawObject, nil
 	}
 
 	for _, objVersion := range obj.History {
 		if objVersion.Version == version {
-			return obj, objVersion, nil
+			copy := &object.RawObject{
+				UID:        obj.UID,
+				Kind:       obj.Kind,
+				Created:    obj.Created,
+				CreatedBy:  obj.CreatedBy,
+				Modified:   objVersion.Modified,
+				ModifiedBy: objVersion.ModifiedBy,
+				ETag:       objVersion.ETag,
+				Version:    objVersion.Version,
+
+				// Not in the history object
```

Comment:
i do not understand what this comment refers to - the `objVersion` _is_ the history object.. right? that is: https://github.com/grafana/grafana/blob/85b965cbec21637fe062572eefa8ca8f10fc102b/pkg/services/store/object/dummy/dummy_server.go#L20-L24

or is this comment referring to the gRPC `ObjectVersionInfo`?